src/index.json

```diff
@@ -328,6 +328,51 @@
                     "summary": "An Azure CLI Extension to manage appservice resources",
                     "version": "0.2.8"
                 }
+            },
+            {
+                "filename": "webapp-0.2.9-py2.py3-none-any.whl",
+                "sha256Digest": "f75b3e09cd3d99a0aaf47d9cf1f3792f6016bbbab4815d42afe6c55e5e306e1f",
+                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.9-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.46",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
+                        "Programming Language :: Python :: 2.7",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "sisirap@microsoft.com",
+                                    "name": "Sisira Panchagnula, Lukasz Stempniewicz",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions"
+                            }
+                        }
+                    },
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "webapp",
+                    "summary": "An Azure CLI Extension to manage appservice resources",
```

Comment:
Please see the [summary guidelines ](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md) for guidance on writing your extension summary.

Something along the lines of  "Additional commands for managing appservice resources", "Support for managing appservice resources" or "Preview appservice management commands" would be good.